README.md

```diff
@@ -478,6 +501,9 @@ This property previously set the frequency with which Quill polled the DOM for c
 `bounds`
 : Selector or DOM element used by Quill to constrain position of popups. Defaults to `document.body`.
 
+`children`
```

Comment:
I removed dangerouslySetInnerHtml from the examples; if you have another fix we can catch it on another PR.